velox/common/base/CompareFlags.h

```diff
@@ -43,4 +43,16 @@ struct CompareFlags {
   }
 };
 
+constexpr const char* NullHandlingModeToStr(
+    CompareFlags::NullHandlingMode nullHandlingMode) noexcept {
+  switch (nullHandlingMode) {
+    case CompareFlags::NullHandlingMode::NoStop:
+      return "NoStop";
+    case CompareFlags::NullHandlingMode::StopAtNull:
+      return "StopAtNull";
+    case CompareFlags::NullHandlingMode::StopAtRhsNull:
+      return "StopAtRhsNull";
```

Comment:
```
s/NullHandlingModeToStr/nullHandlingModeToStr/
```
Add default? And move implementation to cc?

Reply:
There is only a .h file